File: Code/Disney/disney.xBandController/src/linux/IDMSLib/src/main/java/com/disney/xband/idms/lib/model/GuestPartyAssign.java
package com.disney.xband.idms.lib.model;

import javax.xml.bind.annotation.XmlElement;
import javax.xml.bind.annotation.XmlRootElement;

@XmlRootElement
public class GuestPartyAssign {

	private long guestId;
	
	private long partyId;
	
	public GuestPartyAssign() {}

	@XmlElement(name="guestId")
	public long getGuestId() {
		return guestId;
	}

	@XmlElement(name="guestId")
	public void setGuestId(long guestId) {
		this.guestId = guestId;
	}

	@XmlElement(name="partyId")
	public long getPartyId() {
		return partyId;
	}

	@XmlElement(name="partyId")
	public void setPartyId(long partyId) {
		this.partyId = partyId;
	}
	
	
	
}

File: Code/Disney/disney.xBandController/src/windows/Test/Disney.xBand.IDMS/Parties/GuestPartyAssign.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Runtime.Serialization;

namespace Disney.xBand.IDMS.Parties
{
    [DataContract]
    public class GuestPartyAssign
    {
        [DataMember(Name = "guestId")]
        public long GuestID { get; set; }

        [DataMember(Name = "partyId")]
        public long PartyID { get; set; }
    }
}
